.drone.yml

```diff
@@ -41,4 +37,50 @@ pipeline:
       - 'latest'
     when:
       event: push
-      branch: master
\ No newline at end of file
+      branch: master
+
+  e2e_test:
+    image: '${DOCKER_REPO}/${DOCKER_PATH}/${NAME}'
+    secrets: [ service_now_username, service_now_password ]
+    description: 'End-to-end test for the ServiceNow drone plugin. Build #${DRONE_BUILD_NUMBER}'
+    when:
+      event: push
+      branch: master
+
+  notify_slack_success:
```

Comment:
Not sure whether this should go to slack. (Or at least not the same slack channels as our current ones.)

Currently we only alert to slack on deployments.

Reply:
Yes I wasn't sure about this.